### ISSUE TITLE
Can't update settings due to `payment pointer` error.

### ISSUE THREAD

**Issue by ludwiczakpawel** [forem/forem#11784]
**Describe the bug**

I can't update settings due to some error regarding `Payment pointer`.

**To Reproduce**

1. Go to /settings
2. Update anything and try to save
3. **Boom**. `Error: Payment pointer is invalid.`

**Expected behavior**

No **boom**.

**Screenshots**

Video: https://d.pr/v/DyzVIh

**Additional context**

- Seems like it happens on every `/settings/*` subpage.
- Seems like it happens only on prod. I can't repro that locally.

**Comment by kuei0221** [comment]
hi @ludwiczakpawel, I think this related to the PR I made (#11702). and this has added a syntax check to the payment_pointer.

Could you describe what your payment pointer looks like? This error might because your payment pointer now considers invalid. I have reproduced the error with the invalid pointer but nothing happens with a valid one.
The valid pattern will look like this: `$example.com/value`.

**Comment by benhalpern** [comment]
It seems like a "blank" submission is failing...

**Comment by benhalpern** [comment]
..... Okay @ludwiczakpawel I determined that you had an invalid payment pointer of `""`.... And it has been resolved. I fixed it in DEV. If we suspect other communities may have lingering data issues we should have a data update script to fix it.

This also speaks to a scenario I'd love for us to have solutions for in general... Which is to safely "clean up" malformed data as an ongoing process in the longtail of Forems.
